File: app/src/main/java/com/example/karma/response/userImage/Result.java

package com.example.karma.response.userImage;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Result {

    @SerializedName("img")
    @Expose
    private String img;

    public String getImg() {
        return img;
    }

    public void setImg(String img) {
        this.img = img;
    }

}

File: app/src/main/java/com/example/karma/responseModel/updateDefaultPaymentType/Result.kt
package com.example.karma.responseModel.updateDefaultPaymentType

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Result {
    @SerializedName("_id")
    @Expose
    var id: String? = null

    @SerializedName("appId")
    @Expose
    var appId: String? = null

}